app/controllers/garage/docs/resources_controller.rb

```diff
@@ -1,5 +1,4 @@
 require 'oauth2'
-require 'http_accept_language'
```

Comment:
`http_accept_language` 2.0.x is implemented by rack middleware, and the middleware is included by calling `ActiveSupport.on_load` in `Rails::Railtie.initializer` block.

* https://github.com/iain/http_accept_language/blob/master/lib/http_accept_language/middleware.rb
* https://github.com/iain/http_accept_language/blob/master/lib/http_accept_language/railtie.rb#L6

So, we need to call `require 'http_accept_language'` on Rails initialization phase.
This is the reason the one `require` is moved.